lib/common.bash

```diff
@@ -73,7 +79,11 @@ extract_kata_env(){
 # Checks that processes are not running
 check_processes() {
 	extract_kata_env
-	general_processes=( ${PROXY_PATH} ${HYPERVISOR_PATH} ${SHIM_PATH} )
+	if [ "$KATA_HYPERVISOR" == firecracker ]; then
```

Comment:
I'll prefer to make this more generic as this is the case when `vsock` is enabled.
What about check if `vsock` is enabled and then do not check for the proxy process. 
You can use this: https://github.com/kata-containers/tests/blob/master/integration/stability/soak_parallel_rm.sh#L64-L72